index.js

```diff
@@ -23,9 +23,7 @@ module.exports = function(app) {
         app.use(express.static(path.join(__dirname, 'app')));
 
         viewsDirectory += '/app/views';
-    }
-
-    if ('production' === env) {
+    } else {
```

Comment:
Do you think we do need to have these two modes ? From my usage, the only difference I expect between dev and prod is the live-reloading, but maybe specifying  a particular cache behaviour is needed in dev.